client/src/components/onboarding/Choose_Tags.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import "../welcome.css";
+import Paper from "@material-ui/core/Paper";
+import SingleTag from "./SingleTag";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: "flex",
+    justifyContent: "center",
+    flexWrap: "wrap",
+    padding: theme.spacing(0.5)
+  },
+  chip: {
+    margin: theme.spacing(0.5)
+  }
+}));
+
+const initialFounderTags = [
+  { key: 0, label: "Black", selected: false },
+  { key: 1, label: "LBGTQ", selected: false },
+  { key: 2, label: "Asian", selected: false },
+  { key: 3, label: "Women", selected: false },
+  { key: 4, label: "Veteran", selected: false },
+  { key: 5, label: "Student", selected: false },
+  { key: 6, label: "Native American", selected: false }
+];
+
+const initialCompanyTags = [
+  { key: 7, label: "Social Mission", selected: false },
+  { key: 8, label: "Agriculture and Biotech", selected: false },
+  { key: 9, label: "AI and Machine Learning=", selected: false },
+  { key: 10, label: "Entertainment", selected: false },
+  { key: 11, label: "Clean and Renewable Energy", selected: false },
+  { key: 12, label: "Health and mental wellness", selected: false },
+  { key: 13, label: "Food and Drink", selected: false },
+  { key: 14, label: "Financial Serices", selected: false },
+  { key: 15, label: "Ecommerce", selected: false },
+  { key: 16, label: "Internet of things", selected: false },
+  { key: 17, label: "Social and Lifestyle", selected: false }
+];
```

Comment:
I would consider building these two objects dynamically from the back end endpoints. That way if the data changes in the future, it will populate automatically. Use these functions to get your data from the backend
or use the ENV API


```
const getElegibility = async () => {
        return await axios('https://startup-grant-database-staging.herokuapp.com/api/elegibility');
    };

    const getCategories = async () => {
        return await axios('https://startup-grant-database-staging.herokuapp.com/api/categories');
    };
```